cypress/local-only/support/pages/start-a-case.ts

```diff
@@ -7,7 +7,7 @@ export const fillInAndSubmitForm = () => {
 
   // wizard step 1
   cy.get('input#stin-file').should('be.enabled').attachFile(w3Dummy);
-  cy.get('[data-testid="upload-file-success"]');
+  cy.get('[data-testid^="upload-file-success"]');
```

Comment:
This comment was originally

> Is this a stray character?

but then I learned about the "starts with" CSS selector! Neat!